functions/helpFunctions.tsx

```diff
@@ -18,34 +19,24 @@ export interface AnswerType {
   taskNumber: number;
 }
 
-export function insperaDataToTextboxObject(
-  insperaData: any,
-  questionNumber: number
-) {
-  const textboxData: AnswerType[] = [];
-  if (
-    questionNumber < 1 ||
-    questionNumber >
-      insperaData.ext_inspera_candidates[0].result.ext_inspera_questions.length
-  ) {
-    return textboxData;
+export const convertToNumber = (n: any) => {
+  if (typeof n == 'number') {
+    return n;
   }
-  insperaData.ext_inspera_candidates.map((candidate: any) => {
-    textboxData.push({
-      assessmentId: uuidv4(),
-      answer: replaceUndefined(
-        candidate.result.ext_inspera_questions[questionNumber - 1]
-          .ext_inspera_candidateResponses[0].ext_inspera_response
-      ),
-
-      candidateId: parseInt(candidate.result.ext_inspera_candidateId),
-      maxPoints:
-        candidate.result.ext_inspera_questions[questionNumber - 1]
-          .ext_inspera_maxQuestionScore,
-      taskNumber: questionNumber,
-    });
-  });
-  return textboxData;
```

Comment:
Also not related to this PR, just an unused function, as we discussed earlier :)